.github/workflows/docs.yml

```diff
@@ -380,7 +380,7 @@ jobs:
           # PAY ATTENTION: engine folder is 'doc', not 'docs'
           # cd doc && make html && make latexpdf
           cd doc && make html 2>&1 | tee full.log
-          grep WARNING < full.log > warnings.log
+          grep WARNING < full.log > warnings.log || true
```

Comment:
Please add a comment explaining the goal of this (i.e. does the action fail if the file warnings.log is non-empty or not?)

Reply:
Done